src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx

```diff
@@ -70,6 +71,23 @@ class Issues extends React.Component<Props, State> {
     return discoverView.getResultsViewUrlTarget(orgId);
   }
 
+  getIssuesUrl() {
+    const {version, orgId} = this.props;
+    const {issuesType} = this.state;
+    const {queryParams} = this.getIssuesEndpoint();
+
+    return {
+      pathname: `/organizations/${orgId}/issues/`,
+      query: {
+        ...queryParams,
+        query:
+          issuesType === IssuesType.NEW
+            ? `first-release:${version}`
+            : `release:${version}`,
```

Comment:
Could versions contain spaces? You might want to use `app/utils/tokenizeSearch:stringifyQueryObject()` to handle quoting and escaping.

Reply:
Oh, didn't know that there can be an edge case with spaces in release. I've updated it with `stringifyQueryObject`  👍